site/_posts/2023-06-30-announcing-fusionauth-1-46.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.46
+description: This release includes a number of security fixes, improvements to the device grant, and improves the OIDC discovery endpoint. 
```

Comment:
If the subject is "This release" then "improvements to the device grant"  does not make sense and needs a verb.  If the subject is "This release includes" then "improves the OIDC..."  seems off.